src/main/pg/beeper_dev.c

```diff
@@ -17,7 +17,7 @@
 
 #include "platform.h"
 
-#ifdef BEEPER
+#ifdef USE_BEEPER
```

Comment:
This is matched at the very end of this file, hence another `#ifdef` at line 38 is not necessary